packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsUtils.h

```diff
@@ -53,6 +84,111 @@ struct Snapshot {
   }
 };
 
+inline Float floatFromYogaOptionalFloat(yoga::FloatOptional value) {
+  if (value.isUndefined() || std::isnan(value.unwrap())) {
+    return 0;
+  }
+  return (Float)value.unwrap();
+}
+
+inline Float floatFromYogaBorderWidthValue(const yoga::Style::Length &length) {
+  if (length.unit() == yoga::Unit::Point) {
+    return floatFromYogaOptionalFloat(length.value());
+  }
+  // types percent, and auto are not allowed for border width
+  return 0;
+}
+
+static const int numberOfStringProperties = 2;
+static const char *stringPropertiesNames[2] = {
+    "BackgroundColor",
+    "ShadowColor"};
+
+static const int numberOfNumericProperties = 9;
+static const char *numericPropertiesNames[9] = {
+    "Opacity", // double
+
+    "BorderTopLeftRadius", // int
+    "BorderTopRightRadius",
+    "BorderBottomLeftRadius",
+    "BorderBottomRightRadius",
+
+    "BorderLeftWidth", // float
+    "BorderRightWidth",
+    "BorderTopWidth",
+    "BorderBottomWidth"};
```

Comment:
what about the rest of props, for example transform?

Reply:
I've just added support for transform. There are much more props, that may require some additional consideration - do we want to implement shadow unification for Android and IOS? Also there are props with quantified values (borderStyle is either 'solid' or 'dotted').
I would keep the current number of props for now.